main.py

```diff
@@ -0,0 +1,165 @@
+import random
+from uuid import uuid4
+from datetime import datetime
+
+from mimesis import Person, Hardware, Development
+from mimesis.schema import Schema
+from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String, TIMESTAMP, ForeignKey
+import pandas as pd
+
+from config import username, password, dbname, hostname, path
+
+
+class DataGenerator:
+    """
+
+     Define class DataGenerator that creates datasets with fake date.
+     Methods new_users, new_companies and new_devs create dataframes filled with fake data.
+     Method create_tables creates tables users, companies, development inside MySQL DB.
+     Method df_creation triggers methods new_users, new_companies and new_devs.
+     Method insertion moves data from dataframes to tables inside MySQL DB.
+     Method drop_tables drops created tables from MySQL DB.
+    """
+    def __init__(self):
+        self.users = pd.DataFrame()
+        self.companies = pd.DataFrame()
+        self.devs = pd.DataFrame()
+
+    def new_users(self, n):
+        if n < 1:
+            raise ValueError('n must be positive number')
+        user = Person()
+        ids = list(range(0, ))
+        schema = Schema(schema=lambda: {
+            'id': str(uuid4().int)[:7],
+            'name': user.first_name(),
+            'surname': user.surname(),
+            'age': user.age(),
+            'email': user.email(),
+            'gender': user.gender(),
+            'language': user.language(),
+            'country': user.nationality(),
+            'occupation': user.occupation(),
+            'phone': user.telephone(),
+            'username': user.username(),
+            'added_at': datetime.now(),
```

Comment:
please. use datetime.utcnow() for the records that are expected to be stored in the databases - they should match UTC timezone.
```suggestion
            'added_at': datetime.utcnow(),
```